lib/guest.ts

```diff
@@ -0,0 +1,18 @@
+import { getBuilder } from '@nextcloud/browser-storage'
+
+const browserStorage = getBuilder('public').persist().build()
+
+/**
+ * Get the guest nickname for public shares
+ */
+export function getGuestNickname(): string | null {
```

Comment:
Useful for nickvergessen

Reply:
Is it supposed to be used for only and only public shares?

Reply:
@ShGKme the definition of public share can be argued :see_no_evil: 
We currently use if tor public links. E.g Talk, Files, Polls, Text, Colabora....

Reply:
But not for, e.g., Talk guest user?

Reply:
Maybe just reword to `public pages` as a public share is a public page :sweat_smile:

Reply:
> But not for, e.g., Talk guest user?

hmm, yes? I'll reword anyway :see_no_evil: